pymeasure/instruments/hcp/tc038d.py

```diff
@@ -55,89 +53,83 @@ class TC038D(Instrument):
     byteMode = 4
 
     functions = {'read': 0x03, 'writeMultiple': 0x10,
-                 'writeSingle': 0x06, 'echo': 0x08}
+                 'writeSingle': 0x06, 'echo': 0x08,
```

Comment:
Can this be replaced by `Enum` ?

Reply:
I have not implemented enums yet, as dictionaries mostly do the job natively.
I don't see the advantage of enums, when you have dictionaries. I'll do some research.

Reply:
There's a little better/stricter typing, the access pattern is different (e.g. `functions.READ` instead of `functions['read']` here), which means you'd get autocompletion on enum members, and with `IntFlag` enums, you also get bitwise combination possibilities (if you need that), which I've used to great effect in the past.